lib/mxnet/ndarray.rb

```diff
@@ -310,20 +311,60 @@ def _fill_by(value)
       return value_nd
     end
 
+    def self.reshape_like(lhs, rhs, *args, **kwargs)
+      Ops.reshape_like(lhs, rhs, *args, **kwargs)
+    end
+
+    def zeros_like(*args, **kwargs)
+      Ops.zeros_like(self, *args, **kwargs)
+    end
+
+    def ones_like(*args, **kwargs)
+      Ops.ones_like(self, *args, **kwargs)
+    end
+
+    def broadcast_axes(*args, **kwargs)
+      Ops.broadcast_axes(self, *args, **kwargs)
+    end
+
+    alias broadcast_axis broadcast_axes
+
+    def repeat(*args, **kwargs)
+      Ops.repeat(self, *args, **kwargs)
+    end
+
+    def pad(*args, **kwargs)
+      Ops.pad(self, *args, **kwargs)
+    end
+
+    def swapaxes(*args, **kwargs)
+      Ops.swapaxes(self, *args, **kwargs)
+    end
+
     def ndim
       shape.length
     end
+
     alias rank ndim
 
     def size
       shape.inject(:*)
     end
+
     alias length size
 
     def transpose(axes: nil)
       Ops.transpose(self, axes: axes)
     end
 
+    def T
+      if self.shape.size < 2
+        self
+      else
+        self.transpose
```

Comment:
Use `Ops.transpose(self)` instead of `self.transpose` to reduce the number of method calls.